### PR TITLE
Metadata concept documentation

### PR DESCRIPTION
**Pull request by tiagojsag**
This PRs covers https://gfw.atlassian.net/browse/GTC-728

### CODE REVIEW

File: source/includes/concept/_metadata.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Metadata
+
+Understanding the context of data, empowering users to find and make informed decisions about its suitability is another important goal of the Resource Watch API (RW API). For example a user maybe looking to answer questions like *"Does this dataset provide information about tree cover in my region?"*, *"What are the physical units?"*, *"How was this measured?"*, and *"How should I provide proper attribution to the data provider?"*.
```
Comment (ethanroday) [inline source/includes/concept/_metadata.md]:
```suggestion
Understanding the context of data, empowering users to find and make informed decisions about its suitability is another important goal of the Resource Watch API (RW API). For example, a user may be looking to answer questions like, *"Does this dataset provide information about tree cover in my region?"*, *"What are the physical units?"*, *"How was this measured?"*, and *"How should I provide proper attribution to the data provider?"*.
```

File: source/includes/concept/_metadata.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+These questions imply being able to access summary data about the data, so called [Metadata](https://guides.ucf.edu/metadata/intrometadata). By definition, metadata is always associated with another entity, such as a [dataset](https://resource-watch.github.io/doc-api/index-rw.html#dataset), [layer](https://resource-watch.github.io/doc-api/index-rw.html#layer) or [widget](https://resource-watch.github.io/doc-api/index-rw.html#widget), and may be further classified into different types, such as descriptive, structural, and/or administrative. Many different general and discipline-specific metadata standards and schemas are [available](http://rd-alliance.github.io/metadata-directory/standards/), each with their pros and cons. However, in general, the type of information desired about data is:
```
Comment (ethanroday) [inline source/includes/concept/_metadata.md]:
```suggestion
Answers to these questions can be stored in a resource's [metadata](https://guides.ucf.edu/metadata/intrometadata). By definition, metadata is always associated with another entity, such as a [dataset](https://resource-watch.github.io/doc-api/index-rw.html#dataset), [layer](https://resource-watch.github.io/doc-api/index-rw.html#layer) or [widget](https://resource-watch.github.io/doc-api/index-rw.html#widget), and may be further classified into different types, such as descriptive, structural, and/or administrative. Many different general and discipline-specific metadata standards and schemas are [available](http://rd-alliance.github.io/metadata-directory/standards/), each with their pros and cons. However, most types of metadata capture the following core information:
```

File: source/includes/concept/_metadata.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+- **Title/Name** – Name given to the resource.
+- **Description** – A description of the resource, its spatial, temporal or subject coverage, and its methodology and lineage.
+- **Format** – File format, physical medium, dimensions of the resource, or hardware and software needed to access the data.
+- **Metadata schema** – Description of the metadata to be provided along with the generated data and a discussion of the metadata standards used, including the version of the schema and where the schema can be found.
+- **Identifier** – A unique identification assigned to the resource.
```
Comment (ethanroday) [inline source/includes/concept/_metadata.md]: I would leave this one out...seems unnecessary
```suggestion
```

File: source/includes/concept/_metadata.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+- **Rights Holder** – The entities or persons who hold the rights to the data.
+- **Rights** – Information about the rights held in and over the resource.
+- **Contact information** – Identity of, and means to communicate with persons or entities associated with the data.
```
Comment (ethanroday) [inline source/includes/concept/_metadata.md]:
```suggestion
- **Contact information** – Identity of, and means to communicate with, persons or entities associated with the data.
```

File: source/includes/concept/_metadata.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+### Metadata in the context of the RW API
+
+The approach adopted by the RW API is to try to follow the main guidelines about Metadata standards, whilst offering maximum flexibility by only requiring a limited number of core fields, and allowing the inclusion of unlimited, optional application specific fields.
```
Comment (ethanroday) [inline source/includes/concept/_metadata.md]:
```suggestion
The approach adopted by the RW API is to try to follow the main guidelines about metadata standards, whilst offering maximum flexibility by only requiring a limited number of core fields, and allowing the inclusion of unlimited, optional application-specific fields.
```

File: source/includes/concept/_metadata.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+As we'll see in further details when covering the metadata endpoints, a RW API metadata object has a few structured but optional fields to capture some of the general details described above, while also allowing you to specify your own custom fields for ease of extension and flexibility.
+
+Each metadata object must also be associated with a dataset, widget or layer, and must also identify the language in which it's written, as well as the [application](https://resource-watch.github.io/doc-api/index-rw.html#applications) to which it concerns. This allows a single resource, say a dataset, to have its metadata translated into different languages, and be adjusted to meet the needs of different applications. We'll go into further details on these when we cover the metadata endpoints in their dedicated section.
```
Comment (ethanroday) [inline source/includes/concept/_metadata.md]:
```suggestion
Each metadata object must also be associated with a dataset, widget, or layer, and must also identify the language in which it's written, as well as the [application](https://resource-watch.github.io/doc-api/index-rw.html#applications) it's associated with. This allows a single resource, say a dataset, to have its metadata translated into different languages, and be adjusted to meet the needs of different applications. We'll go into further details on these when we cover the metadata endpoints in their dedicated section.
```

**Review by ethanroday — CHANGES_REQUESTED** [review]
I'd say the tone is perhaps overly formal for my taste, but content-wise this is good to go. Most of my edits are small grammar things or clarifying wording.

**Comment by tiagojsag** [comment]
@ethanroday now that I read it, yes, I agree that the tone is a bit more formal than what we've used elsewhere. I've rewritten it so that it fits better with the more pragmatic language we've used previously, let me know if you agree it's better now. And I've merged your grammar improvements, thanks for those :)

**Review by ethanroday — APPROVED** [review]

**Pull request merged**